extensions/Lily/Data.js

```diff
@@ -1,3 +1,9 @@
+// Name: List Tools
+// ID: lmsData
+// Description: An assortment of new was to interact with lists.
```

Comment:
Minor spelling mistake, try again. 🗿

Reply:
Bro I was just going to copy the description from the extended gallery and I am NOT using **that** description